src/constructs/aws/StaticWebsite.ts

```diff
@@ -17,7 +19,18 @@ export class StaticWebsite extends StaticWebsiteAbstract {
         protected readonly configuration: CommonStaticWebsiteConfiguration,
         protected readonly provider: AwsProvider
     ) {
-        super(scope, id, configuration, provider);
+        // we need to build specific bucket props to pass to the abstract static website constructor
+        const bucketProps: BucketProps = {
+            // Enable static website hosting
+            websiteIndexDocument: "index.html",
+            websiteErrorDocument: configuration.errorPage,
+            // public read access is required when enabling static website hosting
+            publicReadAccess: true,
```

Comment:
as stated in https://github.com/getlift/lift/issues/192#issuecomment-1111206102, it was actually an improvement not to have public access on the assets within S3. Is there another solution ? Like using CF function instead ?

Reply:
It would be an improvement indeed but I couldn't make it work :/
Also since a static website has no authentication, it is not dramatic to allow public read access. See https://docs.aws.amazon.com/AmazonS3/latest/userguide/WebsiteAccessPermissionsReqd.html